students/josh_embury/lesson03/mailroom.py

```diff
@@ -0,0 +1,79 @@
+lst_donor_table = [
+['Henry Michalson',10,500,25],
+['Phil Hutch',76],
+['Galileo Humpkins',22000,100,490],
+['Methusela Honeysuckle',18,69,76000],
+['Lavender Goombs',55000,25],
+['test',1]
+]
+def showMenu():
```

Comment:
All function names in this code should be snake_case: show_menu

Reply:
updated